[MIG][v10] mail_digest

simahawk opened the pull request:
ported and improved some bits.
/cc @gurneyalex

simahawk committed:
[add] mail_digest

oca-transbot committed:
OCA Transbot updated translations from Transifex

simahawk committed:
[MIG][v10] mail_digest

ecino reviewed (approved):
I tested it and it works well. The screenshot in the README seems broken :
![image](https://user-images.githubusercontent.com/8435180/33985545-3b95f1f8-e0bb-11e7-822d-82c1de8b17e4.png)
Would you mind giving me back a review? https://github.com/OCA/social/pull/185

simahawk commented:
I'll try fixing the screenshot, tnx

simahawk committed:
[imp] mail_digest: related state is readonly

simahawk committed:
[imp] mail_digest: enforce template domain to type=qweb

simahawk committed:
[imp] mail_digest: fallback to default website

Depending on the request  might fail
nevertheless the website should be there to be found.
Fallback to search and use it if possible for digest' site name

simahawk committed:
mail_digest: upgrade to 10.0.1

simahawk committed:
[fix] mail_digest: manifest website

simahawk committed:
[fix] mail_digest: move `template_id` to `digest_template_id`

When creating digest records within the context of mail composer
(and possibly other contexts) you'll have a `default_template_id`
key in the context which is going to override our safe default.

This is going to break email generation because the template
will be completely wrong and unpredictable. Lesson learned :)

simahawk commented:
@ecino image on odoo is broken because of this https://github.com/odoo/odoo/issues/8300
I gave up as I'd prefer to keep it working on github
